LICENSE

```diff
@@ -1,4 +1,4 @@
-MIT License
+MIT (C) TRON-US
```

Comment:
For this file, could you change Line 3 instead? thanks.

Reply:
ok